datafusion/expr/src/utils.rs

```diff
@@ -653,13 +653,13 @@ where
             if !(exprs.contains(expr)) {
                 exprs.push(expr.clone())
             }
-            // stop recursing down this expr once we find a match
+            // Stop recursing down this expr once we find a match
             return Ok(TreeNodeRecursion::Jump);
         }
 
         Ok(TreeNodeRecursion::Continue)
     })
-    // pre_visit always returns OK, so this will always too
+    // Pre_visit always returns OK, so this will always too
```

Comment:
This looks like a verbatim name, so maybe keep it as is?

"... so this will always return OK too"